### PR TITLE
Update K8s helm chart documentation

### PR DESCRIPTION
**Pull request by madanadit**
- Move shortCircuit policy from fuse to worker helm section
- Fix helm args for Docker entrypoint
- Add back RocksDB helm config

### CODE REVIEW

**Comment by madanadit** [comment]
jenkins, test this please.

File: integration/kubernetes/helm-chart/alluxio/values.yaml [hunk 1]
```diff
@@ -9,67 +9,53 @@
...
+## Master ##
 
 master:
   count: 1 # For multiMaster mode increase this to >1
-  replicas: 1
-  args:
-    - master
-    - -no-format
-  # properties for the master components
+  replicas: 1 # Do NOT modify
```
Comment (jiacheliu3) [inline integration/kubernetes/helm-chart/alluxio/values.yaml]: Do we need a reason for "Do NOT modify" comments?

File: integration/kubernetes/multiMaster-embeddedJournal/master/alluxio-master-statefulset.yaml.template [hunk 2]
```diff
@@ -71,7 +71,9 @@ spec:
               cpu: 1
               memory: 1G
           command: ["/entrypoint.sh"]
-          args: ["master-only", "--no-format"]
+          args:
```
Comment (jiacheliu3) [inline integration/kubernetes/multiMaster-embeddedJournal/master/alluxio-master-statefulset.yaml.template]: Do we explicitly prefer this expression over the `["a", "b"]` way?

File: integration/kubernetes/helm-chart/alluxio/values.yaml (hunk 1 above)
Comment (madanadit) [reply integration/kubernetes/helm-chart/alluxio/values.yaml]: count vs replicas was confusing to me. so i felt like i needed to mention which one to modify. ideally i would remove it from `values.yaml` altogether

File: integration/kubernetes/multiMaster-embeddedJournal/master/alluxio-master-statefulset.yaml.template (hunk 2 above)
Comment (madanadit) [reply integration/kubernetes/multiMaster-embeddedJournal/master/alluxio-master-statefulset.yaml.template]: nope, i just made it consistent w/ what we had in the fuse template previously. its slightly easier to add new args in `config.yaml` using the newline way

File: integration/kubernetes/helm-chart/alluxio/values.yaml (hunk 1 above)
Comment (jiacheliu3) [reply integration/kubernetes/helm-chart/alluxio/values.yaml]: Yea that's true. Maybe mention in the comment that this controls the number of replicas in the StatefulSet and mention in the comment for `count` that `count` controls the number of masters?

File: integration/kubernetes/multiMaster-embeddedJournal/master/alluxio-master-statefulset.yaml.template (hunk 2 above)
Comment (jiacheliu3) [reply integration/kubernetes/multiMaster-embeddedJournal/master/alluxio-master-statefulset.yaml.template]: SGTM

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Pull request merged**